lib/BackbeatMetadataProxy.js

```diff
@@ -105,6 +105,58 @@ class BackbeatMetadataProxy extends BackbeatTask {
         });
     }
 
+    headObject(params, log, cb) {
```

Comment:
Why not `headLocation`, to disambiguate better between doing a HEAD on the zenko object and a HEAD on the object location itself?